packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -337,6 +338,10 @@ class InkResponse extends StatelessWidget {
   /// Called when the user taps down this part of the material.
   final GestureTapDownCallback? onTapDown;
 
+  /// Called when the user releases a tap that was started on this part of the 
```

Comment:
Just a minor analysis fix.

```suggestion
  /// Called when the user releases a tap that was started on this part of the
```